packages/server/lib/util/routes.js

```diff
@@ -1,8 +1,8 @@
 const _ = require('lodash')
 const UrlParse = require('url-parse')
-const konfig = require('../konfig')
 
-const apiUrl = konfig('api_url')
+const app_config = require('../../config/app.json')
+const apiUrl = app_config[process.env.CYPRESS_KONFIG_ENV || process.env.CYPRESS_INTERNAL_ENV || 'development'].api_url
```

Comment:
Is CYPRESS_KONFIG_ENV exposed at all? Wondering if we should rename/get rid of it if we're no longer using konfig. Seems like it might cause more confusion than anything.

Reply:
I don't see it documented anywhere for external use but we do use it to control sending our results to the dashboard when we run our tests. Thus, I am just going to rename it to `CYPRESS_CONFIG_ENV`.

Reply:
renamed [ed83366](https://github.com/cypress-io/cypress/pull/23988/commits/ed833667d8ffad4e78934a4c523eb60951c4f6ec)